vllm/model_executor/layers/quantization/utils/w8a8_utils.py

```diff
@@ -12,7 +12,7 @@
 
 def sparse_cutlass_supported() -> bool:
     # sparse cutlass is not supported on Rocm
```

Comment:
Probably should update the comment (or just remove it, imo the if statement is pretty self-explanatory...)